plasmapy/particles/tests/test_particle_class.py

```diff
@@ -10,6 +10,8 @@
 from astropy.constants import c, e, m_e, m_n, m_p
 
 from plasmapy.particles import json_load_particle, json_loads_particle, molecule
+from plasmapy.particles._isotopes import data_about_isotopes
+from plasmapy.particles._special_particles import particle_zoo
```

Comment:
If we adopt what I mentioned earlier about defining `particle_zoo` in the particles `__init__.py` then this import would need to reflect that.